toolchest_client/api/query.py

```diff
@@ -44,7 +44,8 @@ class Query:
     # Max number of retries on status check timeouts.
     RETRY_STATUS_CHECK_LIMIT = 5
 
-    def __init__(self, stored_output=None, is_async=False, pipeline_segment_instance_id=None):
+    def __init__(self, stored_output=None, is_async=False, pipeline_segment_instance_id=None,
+                 streaming_enabled=False, stored_streaming_client=None):
```

Comment:
Why `stored_streaming_client` instead of the same name as the class attribute + name as it's passed as a parameter (`streaming_client`)?

Reply:
No strong reason, I'll refactor

Reply:
Actually, it's to match the syntax of `stored_output`, but since `self.streaming_client` doesn't generate a new `StreamingClient` object if it isn't given, there's no reason to call it anything different, so I'll go ahead with the refactor to `streaming_client`.